lib/src/widgets/auth_card.dart

```diff
@@ -125,23 +126,23 @@ class AuthCardState extends State<AuthCard> with TickerProviderStateMixin {
     _routeTransitionController.dispose();
   }
 
-  void _switchRecovery(bool recovery) {
+  void _switchRecovery(int fromIndex, bool forward) {
```

Comment:
Why don't you just use `_pageIndex` instead of passing `fromIndex` (which also equal to `_pageIndex` value)?
Also change the method to _switchPage

Reply:
Thanks, I agree and I'll make this change... A potential suggestion for later -- we may not need to keep our own _pageIndex variable. We can call _pageController.page.round() to get the same info.

Reply:
Done